### PR TITLE
utils/storage.js: return early

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: src/js/utils/storage.js [hunk 1]
```diff
@@ -2,25 +2,24 @@ export let idbKeyval = (() => {
   let db;
 
   function getDB() {
-    if (!db) {
-      db = new Promise((resolve, reject) => {
-        const openreq = indexedDB.open('svgo-keyval', 1);
+    if (db) return db;
```
Comment (TrySound) [inline src/js/utils/storage.js:5]: So db will never be cached?

Comment (TrySound) [reply src/js/utils/storage.js:5]: I'd say the code is fine as is. Extra nesting is not always bad.

Comment (jakearchibald) [reply src/js/utils/storage.js:5]: I generally prefer early returns, but yeah, when it comes to caching I prefer:

```js
function whatever() {
  if (notCached) addToCache();
  return getFromCache();
}
```

To:

```js
function whatever() {
  if (cached) return getFromCache();
  addToCache();
  return getFromCache();
}
```

The latter feels more prone to error, like changing how one of the returns works, or forgetting the `addToCache` bit (which is what's happening in this PR).

**Comment by jakearchibald** [comment]
Going to close this one as I don't think it needs changing. We could switch to [idb-keyal](https://github.com/jakearchibald/idb-keyval) at some point, but doesn't really seem necessary.

**Comment by XhmikosR** [comment]
To clarify, my intention was to keep setting the `db` variable. I accidentally removed the assignment.

**Pull request closed without merging**